spark-jobs/src/main/scala/za/co/absa/enceladus/common/config/PathConfig.scala

```diff
@@ -15,4 +15,4 @@
 
 package za.co.absa.enceladus.common.config
 
-case class PathConfig(inputPath: String, outputPath: String)
+case class PathConfig(inputPath: String, outputPath: String, standardizationPath: Option[String] = None)
```

Comment:
Can we get some documentation as to what this represents, or for what purpose it was created?

Reply:
I will document it since I'm not sure this approach is the right one

Reply:
I would say that having `rawPath` and `publishPath` would mean less reuse and more of the conditional logic, but if it's easier to read than I can do it

Reply:
I don't think there would be much, actually any conditional logic. Standardization logic would use only `rawPath` and `standardizationPath`, Conformance logic would use `standardizationPath` abd `rawPath`.
Only some logging might need to move from `CommonJobExecution` higher.
On the other hand `getPathCfg` could move to to `CommongJobExecution`, well in a way. I would use the defualt filing in `CommonJobExecution`. Override in higher classes as such: in case of `rawPathOverride`/`publishPathOverride` is defined copy the result of ancestor result with the override value.

Reply:
When I asked for documentation I meant like "This case class represents the paths of input data, intermediary data and result data of the Enceladus spark jobs"